.github/workflows/sims.yml

```diff
@@ -39,8 +39,8 @@ jobs:
         with:
           go-version-file: 'go.mod'
         if: env.GIT_DIFF
-      - run: NUM_BLOCKS=100 make sim-app
-        if: env.GIT_DIFF
+#      - run: NUM_BLOCKS=100 make sim-app
+#        if: env.GIT_DIFF
```

Comment:
Experiencing unexpected errors with simulations specifically such as:

https://github.com/regen-network/regen-ledger/actions/runs/5214786909/jobs/9411583628

Not sure what is causing this but not an issue on `main`. Still investigating but might need to simply disable for now as this seems specific to sims (i.e. unit tests and integration tests are not affected). We also have an issue in the sims nightly flow and in general sims need to be updated.